examples/advanced/backup.yaml

```diff
@@ -0,0 +1,260 @@
+    ## Prefix specifies the data path.
+    ## If set, the value is used to make up the path of the remote storage: `gcs://${.spec.gcs.bucket}/${.spec.gcs.prefix}/backupName`. 
+    # prefix: ""
+  
+  # local:
+    ## Volume presents the persistent volume configuration.
+    ## specify volume types that are supported by Kubernetes, Ref: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes
+    # volume:
+      # name: nfs
+      # nfs:
+        # server: 192.168.0.2
+        # path: /nfs
+    ## VolumeMount is the persistent volume mount configuration.
+    # volumeMount:
+      # name: nfs
+      # mountPath: /nfs
+    ## Prefix of the data path.
+    ## If set, the value is used to make up the storage path of the persistent volume: `local://${.spec.local.volumeMount.mountPath}/${.spec.local.prefix}/`.
+    # prefix: ""
+
+
+  ## The storageClassName of the persistent volume for Backup data storage.
+  ## Defaults to Kubernetes default storage class.
+  # storageClassName: ""
+
+  ## StorageSize is the PV size specified for the backup operation.
+  ## This value must be greater than the size of the TidbCluster to be backed up.
+  ## Defaults to 100Gi.
+  storageSize: "100Gi"
+
+
+  ## BR configuration.
+  br:
+    ## Cluster specifies name of TidbCluster to be backed up.
+    cluster: "advanced-tidb"
+    ## Namespace specifies namespace of TidbCluster to be backed up.
+    clusterNamespace: "default"
+    ## Deprecated from BR v4.0.3.
+    ## Please use `Spec.TableFilter` instead. DB is the specific DB which will be backed up or restored.
+    # db: ""
+    ## Deprecated from BR v4.0.3.
+    ## Please use `Spec.TableFilter` instead. Table is the specific table which will be backed up or restored.
+    # table: ""
+    ## LogLevel is the log level. Defaults to `info`.
+    # logLevel: "info"
+    ## StatusAddr is the HTTP listening address for the status report service. Defaults to empty.
+    # statusAddr: ""
+    ## Concurrency is the size of thread pool on each node that execute the backup task.
+    ## Defaults to 4.
+    concurrency: 4
+    ## RateLimit is the rate limit of the backup task, MB/s per node.
+    ## If set to 4, the speed limit is 4 MB/s.The speed limit is not set by default.
+    # rateLimit: 0
+    ## TimeAgo presents back up the data before `timeAgo`, e.g. 1m, 1h. Defaults to empty.
+    # timeAgo: 1m
+    ## Checksum specifies whether to verify the files after the backup is completed.
+    ## Defaults to `true``.
+    # checksum: true
+    ## SendCredToTikv specifies whether the BR process passes its AWS or GCP privileges to the TiKV process.
+    ## Defaults to `true``.
+    # sendCredToTikv: true
+    ## OnLine specifies whether online during restore. Defaults to false.
+    # onLine: false
+    ## Options specifies the extra arguments that BR supports. These options has highest priority.
+    # options: []
+
+  ## DumplingConfig is the configs for dumpling
```

Comment:
How about adding a link to the dumpling doc?

Reply:
The suggestions above have been fixed, PTAL.